provisioning/device/src/Models/ProvisioningErrorDetails.cs

```diff
@@ -36,32 +33,6 @@ public class ProvisioningErrorDetails
         /// Additional information.
         /// </summary>
         [JsonProperty(PropertyName = "info", NullValueHandling = NullValueHandling.Ignore)]
-        public Dictionary<string, string> Info { get; private set; } = new Dictionary<string, string>();
-
-        /// <summary>
-        /// Time stamp (in UTC).
-        /// </summary>
-        public string TimestampUtc { get; set; }
```

Comment:
No reference and no JsonProperty for deserialization, and no similar property in Java.